### PR TITLE
Bump copyright year to 2022

### PR DESCRIPTION
**Pull request by upsj**
Happy new year :)

### CODE REVIEW

**Comment by upsj** [comment]
format!

**Review by pratikvn — APPROVED** [review]
Happy new year!

**Review by greole — COMMENTED** [review]
Happy new year!

**Review by yhmtsai — APPROVED** [review]
Happy new year

File: dev_tools/scripts/config [hunk 1]
```diff
@@ -36,7 +36,7 @@
 - "core/test/base/(extended_float|iterator_factory)"
     - RemoveTest: "true"
 - "core/test/base/allocator"
-    - FixInclude: "ginkgo/core/base/allocator.hpp"
```
[the remark this answers is not in the document]
Comment (upsj) [reply dev_tools/scripts/config]: This was a change we forgot to revert in another PR that for whatever reason didn't cause an error, so I added it here to make sure it doesn't happen in the future.

**Pull request merged**